File: _Contabilidad/MantenimientoCtasContables/BuscaCta.vb
Option Strict Off
Option Explicit On

Public Class BuscaCta
    Public Function BuscaCtaCtb(ByRef Nombre As String, ByRef TipoMovimiento As String) As String
        Dim prog As New BuscaCtaContab
        prog.IniciaCtas(Nombre, TipoMovimiento)
        BuscaCtaCtb = CodigoCta
        Nombre = NombreCta
    End Function
    Public Sub ImprimePlanCtas()
        Dim prog As New CTBP21
        prog.ShowDialog()
        prog = Nothing
    End Sub

    Public Sub MantenCtas()
        Dim prog As New CTBP01
        prog.ShowDialog()
        prog = Nothing
    End Sub

End Class

File: _Contabilidad/MantenimientoCtasContables/BuscaCta.cs

namespace MantCtb
{
    public class BuscaCta
    {
        public string BuscaCtaCtb(ref string Nombre, ref string TipoMovimiento)
        {
            string BuscaCtaCtbRet = default;
            var prog = new BuscaCtaContab();
            BuscaCtaCtbRet = "";
            prog.IniciaCtas(ref Nombre, ref TipoMovimiento);
            BuscaCtaCtbRet = OperacionesBuscaCta.CodigoCta;
            Nombre = OperacionesBuscaCta.NombreCta;
            prog = null;
            return BuscaCtaCtbRet;
        }

        public void ImprimePlanCtas()
        {
            var prog = new CTBP21();
            prog.ShowDialog();
            prog = null;
        }

        public void MantenCtas()
        {
            var prog = new CTBP01();
            prog.ShowDialog();
            prog = null;
        }
    }
}